### PR TITLE
Feat/update styling

### PR DESCRIPTION
**Pull request by rockse**
**Problem**
The button looks a little odd as it doesn't follow GitHub design system. We could make it look more like a GitHub component. 

**Solution**
This PR updates the styling of `button` to the [Github primer](https://primer.style/css/components/labels) design system. We are using CSS from `labels` to adapt our component.

I have also formatted all the files using prettier with the default config from vscode plugin

**Screenshots**
Screenshot 2022-11-13 at 22 38 01
Screenshot 2022-11-13 at 22 37 30
Screenshot 2022-11-13 at 22 37 36

_PS: I have tested my changes in my chrome browser_

### CODE REVIEW

File: content.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 /* global chrome, PHRASES_TO_HIDE */
 
 (function () {
-  const itemClass = 'TimelineItem';
-  const itemHiddenClass = 'TimelineItem-Hidden';
-  const toggleButtonID = 'tidy-timeline-button';
+  const itemClass = "TimelineItem";
```
Comment (andreacaldera) [inline content.js]: Shall we revert these formatting changes and add prettier config?

Comment (rockse) [reply content.js]: Sure, happy with that :) Do you have the prettier config you used to format the project?

Comment (mcalthrop) [reply content.js]: Agree; single quotes are a defacto JS standard, and used in other M&S repos as well.

File: content.js [hunk 2]
```diff
@@ -154,7 +156,9 @@
     if (newPageUrl !== pageUrl) {
       pageUrl = newPageUrl;
       await new Promise((res) => setTimeout(res, 500));
-      const timeline = document.querySelectorAll('.pull-discussion-timeline')[0];
+      const timeline = document.querySelectorAll(
```
Comment (mcalthrop) [inline content.js]: Could use a `.editorconfig` file as well.

File: styles.css [hunk 3]
```diff
@@ -1,47 +1,55 @@
...
-    bottom: -60px;
-    background: black;
-    color: white;
-    padding: 10px;
-    border: none;
-    display: flex;
-    align-items: center;
-    z-index: 100;
-    transition: bottom 0.4s ease-out;
+  position: fixed;
```
Comment (mcalthrop) [inline styles.css:6]: indentation would be set in `.editorconfig`

File: content.js (hunk 1 above)
Comment (rockse) [reply content.js]: I have added prettier and editor config

**Review by andreacaldera — APPROVED** [review]

**Pull request merged**